src/pages/images/actions/forms/ImportImageFromTarballForm.tsx

```diff
@@ -0,0 +1,157 @@
+import { ChangeEvent, FC, useState } from "react";
+import { useEventQueue } from "context/eventQueue";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { createImageAlias, createImageFromTarball } from "api/images";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { Link, useNavigate } from "react-router-dom";
+import { getImageToTarballBody, humanFileSize } from "util/helpers";
+import ProgressBar from "components/ProgressBar";
+import { UploadState } from "types/storage";
+import Loader from "components/Loader";
+
+interface Props {
+  close: () => void;
+  project: string;
+}
+
+const ImportImageFromTarballForm: FC<Props> = ({ close, project }) => {
+  const eventQueue = useEventQueue();
+  const toastNotify = useToastNotification();
+  const navigate = useNavigate();
+  const [uploadState, setUploadState] = useState<UploadState | null>(null);
+
+  const notifySuccess = () => {
+    const imported = <Link to={`/ui/project/${project}/images`}>imported</Link>;
+    toastNotify.success(<>Image {imported} from file.</>);
```

Comment:
```suggestion
    toastNotify.success(<>Image {imported}.</>);
```
More concise and just as precise?